### PR TITLE
Add mr_test Tests in osd for EC2

### PR DESCRIPTION
**Pull request by lilyeyes**
Add mr_test Tests in osd for EC2: maintenance part + 15SP5 part
[TEAM-7779](https://sd.suse.com/browse/SD-7779) - Enable saptune/mr_test Tests in osd for AWS/EC2

Related MRs:
MR (qem job template): https://gitlab.suse.de/qa-css/openqa_ha_sap/-/merge_requests/433  
MR (15SP5  job template): https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/1174  
MR (15SP5 trigger): https://gitlab.suse.de/qac/trigger-public-cloud/-/merge_requests/138  

- Related ticket: https://sd.suse.com/browse/SD-7779
- Needles: NA
- Verification run:   

**15SP5 part:**   
PAYG: http://openqa.suse.de/tests/10945954#dependencies  
All cases failed on **registration** module:  "zypper -n --gpg-auto-import-keys ref" returns "Warning: There are no enabled repositories defined"  
BYOS: http://openqa.suse.de/tests/10945959#dependencies
The timed out issues are known.

**QEM part:**    
15SP4 (all passed): https://openqa.suse.de/tests/overview?distri=sle&version=15-SP4&build=%3Awhatever%3Atest-15sp4&groupid=477
  15SP3 (all soft failures and timed out issues are known): https://openqa.suse.de/tests/overview?distri=sle&version=15-SP3&build=%3Awhatever%3Atest-15sp3&groupid=477
  15SP2 (all soft failures and timed out issues are known): https://openqa.suse.de/tests/overview?distri=sle&version=15-SP2&build=%3Awhatever%3Atest-15sp2&groupid=477
  15SP1 (all soft failures are known): https://openqa.suse.de/tests/overview?distri=sle&version=15-SP1&build=%3Awhatever%3Atest-15sp1&groupid=477
12sp5 (all soft failures are known): https://openqa.suse.de/tests/overview?distri=sle&version=12-SP5&build=%3Awhatever%3Atest-12sp5&groupid=477
12sp4 (all soft failures are known): https://openqa.suse.de/tests/overview?distri=sle&version=12-SP4&build=%3Awhatever%3Atest-12sp4&groupid=477

### CODE REVIEW

File: data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml [hunk 1]
```diff
@@ -24,8 +24,9 @@ terraform:
     hana_instancetype: "%PUBLIC_CLOUD_INSTANCE_TYPE%"
 
 ansible:
-  hana_urls:
+  az_storage_account_name: "%HANA_ACCOUNT%"
+  az_container_name:  "%HANA_CONTAINER%"
+  hana_media:
```
Comment (mpagot) [inline data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml:29]: Example of a resulting hana_media.yaml is https://openqa.suse.de/tests/10945955/file/qesap_terraform-hana_media.yaml

It is not ok. Please compare it with http://openqaworker15.qa.suse.cz/tests/146995/file/configure-hana_media.yaml that is using https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/sles4sap/qe_sap_deployment/qesap_aws.yaml

`az_sas_token` is missing

Comment (lilyeyes) [reply data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml:29]: Yes, I noticed this. But test module can pass so I did not add "az_sas_token". Why no "az_sas_token" test module still can pass?

Comment (lilyeyes) [reply data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml:30]: Added "az_sas_token".

Comment (mpagot) [reply data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml:30]: > Why no "az_sas_token" test module still can pass?

As mr_test is not calling Ansible at all

File: data/sles4sap/qe_sap_deployment/mr_test_ec2_product.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+provider: "ec2"
```
Comment (mpagot) [inline data/sles4sap/qe_sap_deployment/mr_test_ec2_product.yaml:1]: As it does not come from a setting here, I'd propose to use the `qe-sap-deployment` style name --> **aws**

**Pull request merged**